apps/src/templates/progress/GoogleClassroomShareButton.jsx

```diff
@@ -54,13 +47,6 @@ export default class GoogleClassroomShareButton extends React.PureComponent {
     // Use unique callback names since we're adding to the global namespace
     window[this.onShareStartName()] = this.onShareStart;
     window[this.onShareCompleteName()] = this.onShareComplete;
-
-    // The button callbacks are not supported in IE, so in IE we use click
-    // events to record analytics. The button is rendered in an iframe though,
-    // so to detect the click events we need to use the 'blur' event.
-    if (isIE11) {
```

Comment:
Oh -- I'm guessing this is always evaluating to true (regardless of browser) and should actually have been a function call like `isIE11()` 🤦